plugins/http_plugin/include/eosio/http_plugin/beast_http_session.hpp

```diff
@@ -132,14 +130,16 @@ class beast_http_session : public detail::abstract_conn {
 
       // Request path must be absolute and not contain "..".
       if(req.target().empty() || req.target()[0] != '/' || req.target().find("..") != beast::string_view::npos) {
+         fc_ilog( plugin_state_->logger, "Return bad_reqest:  ${target}",  ("target", std::string(req.target())) );
          error_results results{static_cast<uint16_t>(http::status::bad_request), "Illegal request-target"};
          send_response( fc::json::to_string( results, fc::time_point::maximum() ),
                         static_cast<unsigned int>(http::status::bad_request) );
          return;
       }
 
       try {
-         if(!derived().allow_host(req)) {
+         if(!allow_host(req)) {
+            fc_ilog( plugin_state_->logger, "bad host:  ${HOST}", ("HOST", std::string(req["host"])));
```

Comment:
If this is kept it should be `fc_dlog`, but does it add anything to the `fc_dlog` in `send_response`?